meson.build

```diff
@@ -431,6 +431,7 @@ foreach it : ccs
       ['__builtin_bswap16', '', []],
       ['__builtin_bswap32', '', []],
       ['__builtin_bswap64', '', []],
+      ['__builtin_clzll', '', []],
```

Comment:
If you want this to work correctly you should add the right entry in rz_userconf.h.in, otherwise it does nothing.